sdk/monitor/monitor-query/tests.yml

```
@@ -4,8 +4,4 @@ extends:
     template: /eng/pipelines/templates/stages/archetype-sdk-tests.yml
     parameters:
       PackageName: "@azure/monitor-query"
-      ServiceDirectory: monitor
-      EnvVars:
-        AZURE_CLIENT_ID: $(aad-azure-sdk-test-client-id)
-        AZURE_TENANT_ID: $(aad-azure-sdk-test-tenant-id)
-        AZURE_CLIENT_SECRET: $(aad-azure-sdk-test-client-secret)
+      ServiceDirectory: monitor      
```

```suggestion
      ServiceDirectory: monitor      
      UseFederatedAuth: true
```
This variable will ensure the right credential is used in the live test pipeline.